src/org/zalando/stups/magnificent/external/account.clj

```diff
@@ -0,0 +1,49 @@
+(ns org.zalando.stups.magnificent.external.account
+  (:require [clj-http.client :as http]
+            [org.zalando.stups.friboo.ring :as util]
+            [org.zalando.stups.magnificent.util :refer [defmemoized]]
+            [com.netflix.hystrix.core :refer [defcommand]]))
+
+(defn format-account
+  [account]
+  (update-in
+    (select-keys account
+      [:id
+       :name
+       :disabled
+       :type
+       :members])
+    [:members]
+    (fn [members]
+      (map
+        #(assoc {}
+          :id (:id %)
+          :realm "employees")
+        members))))
+
+(defn condense-account
+  [account]
+  (select-keys account [:id :type]))
```

Comment:
why do we only extract so few properties. I could imagine that, e.g. the `owner`, or the `disabled` information would be very useful here as well.